Cargo.toml

```
@@ -67,6 +67,12 @@ all-features = true
 atomic = "0.5.0"
 bytes = "1"
 futures = "0.3.1"
+# Explicit dependency to be used in `wasm-bindgen` feature
+futures-timer = "3.0.2"
```

Can we have the comment _next_ to the dependency? That way it is a single line and the table can more easily be sorted alphabetically.

```suggestion
futures-timer = "3.0.2" # Explicit dependency to be used in `wasm-bindgen` feature
```